### PR TITLE
fix: Open changelog's link to development commits in an external browser

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Purpose / Description
If you tapped the link for the development version commits in the changelog, instead of opening it in an external browser, it would open it inside the changelog webview, which had the problem of the links being red and didn't offer the full power of a real browser.

image

## Approach
Excludes the commits link from the links to be opened inside the changelog webview

## How Has This Been Tested?

API 25 emulator

[device-2023-06-26-182023.webm](https://github.com/ankidroid/Anki-Android/assets/69634269/aa773755-4e52-46d6-92fd-f325dfe09447)

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [X] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/Info.kt [hunk 1]
```diff
@@ -130,18 +128,20 @@ class Info : AnkiActivity() {
                     ): Boolean {
                         // Excludes the url that are opened inside the changelog.html
                         // and redirect the user to the browser
-                        if (request?.url.toString() in arrayListOf(CHANGE_LOG_URL, GITHUB_COMMITS)) {
+                        val url = request?.url?.toString()
+                            ?: return false
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.kt:132]: Style: should be put on the line above.

File: AnkiDroid/src/main/java/com/ichi2/anki/Info.kt [hunk 2]
```diff
@@ -130,18 +128,20 @@ class Info : AnkiActivity() {
...
                             // snackbar can't be used here as it's a webview and lack coordinator layout
                             UIUtils.showThemedToast(
                                 this@Info,
-                                resources.getString(R.string.no_browser_notification) + request?.url.toString(),
+                                resources.getString(R.string.no_browser_notification) + url,
                                 false
                             )
                         } else {
-                            Intent(Intent.ACTION_VIEW, Uri.parse(request?.url.toString())).apply {
+                            Intent(Intent.ACTION_VIEW, Uri.parse(url)).apply {
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/Info.kt:144]: Style: I don't think `apply {}` is valuable here: `startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(url)))`

**Review by lukstbit — APPROVED** [review]
LGTM
Left some small style issues if you want to address here.

**Review by mikehardy — APPROVED** [review]
I like this

The style issues are minor but not bad suggestions, adopt them I think unless you have a strong disagreement, then merge as soon as CI goes green

**Pull request merged**